scrapy_poet/page_input_providers.py

```diff
@@ -333,3 +335,31 @@ async def __call__(
 
             results.append(item)
         return results
+
+
+class CrawlerStatCollector(StatCollector):
```

Comment:
ScrapyPoetStatsCollector?

Reply:
Although I got it now, why it's called CrawlerStatsCollector: it uses crawler.stats